src/components/Text.js

```diff
@@ -64,7 +67,7 @@ const Text = React.forwardRef(({
 
     return (
         // eslint-disable-next-line react/jsx-props-no-spreading
-        <RNText ref={ref} style={[componentStyle]} {...props}>{children}</RNText>
+        <RNText allowFontScaling={false} ref={ref} style={[componentStyle]} {...props}>{children}</RNText>
```

Comment:
Maybe I am missing something, but why do we set `allowFontScaling={false}` when the props already have the `allowFontScaling` default value set as false?

Reply:
You are right, deleted it from the props.